.github/dependabot.yml 

```diff
@@ -0,0 +1,12 @@
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "06:00"
```

Comment:
This still seems helpful to me -- it would hit early-ish in everyone's day, so they can be reviewed and merged early, and quickly pulled the same day if we discover an issue not found by tests.

Reply:
Agreed!